docs/githubteacher.md

```diff
@@ -0,0 +1,5 @@
+# GitHub Teacher
+
+- **Name:** GitHub Teacher
+- **Occupation:** Teacher
+- **Location:** San Francisco
```

Comment:
Maybe we should be more specific here.  There are lots of San Franciscos in the :earth_americas:.